internal/ratelimit/interval.go

```diff
@@ -0,0 +1,63 @@
+package ratelimit
```

Comment:
this and `interval_test.go` were just moved over from `ratelimits.go` and `ratelimits_test.go` right? were there any changes made besides renaming the structs? github doesn't make it easy to review files that are moved unfortunately

Reply:
Interval limiter was unchanged.